ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/versions/electra/helpers/MiscHelpersElectraTest.java

```diff
@@ -73,4 +87,45 @@ public void isFormerDepositMechanismDisabled_returnsFalseIfNotDisabled() {
 
     assertThat(miscHelpersElectra.isFormerDepositMechanismDisabled(state)).isFalse();
   }
+
+  @Test
+  void consolidatedValidatorsMoreLikelyToPropose() {
+    final int consolidationAmount = 16;
+    final BeaconState state = randomStateWithConsolidatedValidator(consolidationAmount);
```

Comment:
Don't you mind to have a variable or constant to store the validator index. just to keep those "3"s aligned :)

Reply:
sure - thats fine...